tools/ci/compile.sh

```diff
@@ -79,6 +79,11 @@ if [ $EXIT_CODE != 0 ]; then
     exit $EXIT_CODE
 fi
 
+if [[ ${PROFILE} == *"-Dfast"* || ${PROFILE} == *"-Pfast"* ]]; then
```

Comment:
this should also take passed arguments into account 🤔 The duality or args being passed via profile _and_ arguments to compile.sh is a bit annoying.